### PR TITLE
pool.go: Fix golint issues#1953

### PR DESCRIPTION
**Pull request by sarnava1**
Closes openebs/openebs#1953
Signed-off-by: Sarnava Konar <konarsarnava@gmail.com>

**What this PR does / why we need it**: Golint fixes

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes https://github.com/openebs/openebs/issues/1953

**Special notes for your reviewer**:N/A

### CODE REVIEW

**Comment by sarnava1** [comment]
@Akash4927 @satyamz @sonasingh46 @princerachit please review my PR. It is my first contribution to openebs org and my first try at go language. Please do guide me if I have made any mistake. Please review it and merge it if it is okay. Thanks a lot to Hacktoberfest that I got to know of openebs.

**Comment by sarnava1** [comment]
Thanks a lot @Akash4927 for reviewing

**Comment by sarnava1** [comment]
@princerachit @satyamz @sonasingh46 please review my PR so that it can be merged.

File: cmd/cstor-pool-mgmt/pool/pool.go [hunk 1]
```diff
@@ -35,12 +35,15 @@ const (
...
 
+// PoolNamePrefix declares a variable string of type string to store the prefix of pool name
 type PoolNamePrefix string
 
+// PoolPrefix assigns prefix of pool name 
 const (
 	PoolPrefix PoolNamePrefix = "cstor-"
 )
 
+// RunnerVar declares variable of util package to help import and build pool
```
Comment (princerachit) [inline cmd/cstor-pool-mgmt/pool/pool.go]: `// RunnerVar the runner variable for executing binaries.`

File: cmd/cstor-pool-mgmt/pool/pool.go [hunk 2]
```diff
@@ -35,12 +35,15 @@ const (
 //PoolAddEventHandled is a flag representing if the pool has been initially imported or created
 var PoolAddEventHandled = false
 
+// PoolNamePrefix declares a variable string of type string to store the prefix of pool name
```
Comment (princerachit) [inline cmd/cstor-pool-mgmt/pool/pool.go]: `// PoolNamePrefix is a typed string to store pool name prefix `

File: cmd/cstor-pool-mgmt/pool/pool.go [hunk 3]
```diff
@@ -35,12 +35,15 @@ const (
...
 type PoolNamePrefix string
 
+// PoolPrefix assigns prefix of pool name
```
Comment (princerachit) [inline cmd/cstor-pool-mgmt/pool/pool.go]: `// PoolPrefix is prefix for pool name`

**Comment by sarnava1** [comment]
Thank you @princerachit for reviewing. I will do the required changes soon.

**Comment by sarnava1** [comment]
@princerachit @Akash4927 @satyamz @sonasingh46 I have done the changes as asked by @princerachit . Please review my PR.

**Comment by sarnava1** [comment]
Thanks a lot @princerachit @Akash4927 .

**Pull request merged**